### PR TITLE
feat(test): add schema tests

### PR DESCRIPTION
**Pull request by jeremyfiel**
additional schema tests for v1.0.0

### CODE REVIEW

**Comment by jeremyfiel** [comment]
you need to add the actual schema for the tests to work correctly.  the stub has an empty schema so all the failing tests are failing.

**Comment by jeremyfiel** [comment]
reminder :)

**Comment by ralfhandl** [comment]
Why should I add them to my PR?

Wouldn't it be easier to merge my PR first, then you update your PR from `main`, adding a real schema and its tests in one go?

File: tests/v1.0/pass/actions.structured-overlay-example.yaml
Comment (ralfhandl) [inline tests/v1.0/pass/actions.structured-overlay-example.yaml:1]: Why `actions.` here and `actions-` in most of the other test files for actions?

**Comment by ralfhandl** [comment]
Closing this in favor of https://github.com/OAI/Overlay-Specification/pull/94

**Pull request closed without merging**